spec/support/page_objects/check_location_page_object.rb

```diff
@@ -1,18 +1,11 @@
 class CheckLocationPageObject < BasePageObject
```

Comment:
@autotelik @reggieb I've got some reservations about using feature spec + page objects here and maybe we need a meeting about our testing approach in the two apps. I'm glad we are not using feature specs in the engine as specs run *sooo* much faster there. I thought we might test the page content in the engine using controller/vew specs to keep things fast. The page content and workflow in the journey is the responsibility of the engine (unless i18n-overridden in the app, which I don't think we need to do) so should be tested there. Tim S is writing cucumber scenarios which might well do the job of the feature specs in this PR. And we don't want to duplicate testing. So we need to think about what kind of tests we have in which component to keep tests running quickly, avoid duplication, and keep the code as simple as possible and as easy as possible to encourage our successors to write specs. Ug.